packages/shared/src/components/auth/LoginForm.tsx

```diff
@@ -48,6 +48,7 @@ function LoginForm({
     onPasswordLogin(form);
   };
   const [shouldFocus, setShouldFocus] = useState(autoFocus);
+  const [innerEmail, setInnerEmail] = useState(email || '');
```

Comment:
I was thinking, we should be able to just use `ref` on `TextField` below and then just use `textFieldRef.current.value` when calling `onForgotPassword` below. It's clunky but at least we don't introduce additional state just for the sake of it and since field is uncontrolled.

Reply:
I know `TextField` has some ghosts so if you already tried it let's leave it as it is.